src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
Looking at the existing code, the FileSystemProvider has a ton of `#if UNIX` and `#if !UNIX` already because the filesystems are inherently different (beyond just NTFS streams).  I think refactoring this code is an exercise best left for the future.